.github/workflows/main.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+
+jobs:
+  style:
+    runs-on: ubuntu-latest
+    name: Style checks
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.2
+      
+      - name: Install dependencies
+        run: |
+          npm i prettier
+          sudo npm i -g lintspaces-cli
+
+      - name: Prettier
+        run: ./node_modules/prettier/bin-prettier.js --check ./src/* ./src/**/* ./src/**/**/* ./sass/*.scss
+
+      - name: lintspaces
+        run: |
+          lintspaces -nt -d 'spaces' -i 'js-comments' src/*/* src/*.ts view/* manifest-*.json css/popup.css css/import.css
+  build-ffx:
+    runs-on: ubuntu-latest
```

Comment:
Do we need set up new environment for build?

Reply:
No, each key under jobs is run in a separate environment.